prime-router/src/main/kotlin/azure/ApiKeysFunctions.kt

```diff
@@ -174,36 +184,34 @@ SwIDAQAB
 @SecuritySchemes(
     value = [
         SecurityScheme(
-            name = "primeSecurity",
+            name = "primeSecurityOAUTH",
```

Comment:
Did not test this locally since you did a demo on this. Can you explain why the CORS trouble? Is that because we are running locally and this will not be an issue on staging/production?

Reply:
On staging / prod, CORS still needs to be properly configured, CORS configuration should be a common practice for DevOpSec team, in this case, the browser expects a request with Origin header present but there is not:

OKTA doc on the error :

https://support.okta.com/help/s/article/Browser-requests-to-the-token-endpoint-must-use-Proof-Key-for-Code-Exchange?language=en_US